lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -120,50 +125,104 @@ def test_clean_empties(self, server_config, make_logger):
             for r in roots:
                 assert not (r / c).exists()
 
-    def test_create_bad(self, selinux_disabled, server_config, make_logger, tarball):
+    def test_metadata(
```

Comment:
The `tarfile` `extractfile` method will apparently raise `KeyError` if the metadata.log target doesn't exist, and return `None` on unspecified other errors. (It's not entirely clear from the documentation whether it might raise `TarError`?) Those cases ought to be tested, and I don't think they are?